src/assets/styles/global/helper-classes.scss

```diff
@@ -62,7 +62,7 @@
 ============================================================================== */
 .skip-link:focus {
   @include visually-shown(absolute);
-  z-index: $z-index-skip-to-content;
+  z-index: 10000;
```

Comment:
I'm curious... Is there a reason why you hardcoded this value but for you have other variables such as `$placeholder-color-foreground` and `$placeholder-color-foreground`?

I'm not sure if someday we will have 2 or more "skip links" in the future?

Reply:
Right, I see what you mean. Yes, for the time being, the placeholder foreground and background variables are used once or twice so I feel it's not needed. If we are using it more often in the near future, we'll put it back 👍